xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/macros.vm

```diff
@@ -953,16 +953,44 @@ $html
 ###    message
 ###
 
+#macro(genericMessage $text $classAffix $iconName $prettyNameKey $isInline)
```

Comment:
This is definitely missing some documentation: be aware that this macro will become public API. It needs docs, since etc. Unless you think it should only be internal, in which case it should be named `_genericMessage`. But still deserves doc.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3555/commits/6b9cce1fbefb87c115ba6647c3ef851314cccf94 :+1: